### ISSUE TITLE
Location not being recorded where the map is clicked

### ISSUE THREAD

**Issue by Orienteerer** [marchyman/GeoTag#55]
Been using GeoTag successfully for a long time.

I'm now using GeoTag v5.3 on a Mac running Sonoma 14.1.2 Apple (M1 Max chip)

If I select a photo then click on the map, the red pin appears away from the click point - usually further north and outside the view area of the map, so you have to zoom out to find it. 
If I save the geotag data and click back on the photo, the red pin reappears on the map in the same place - but still NOT where I clicked on the map.
If I select a photo and click on the map at a point I'll call X, a red pin appears in the wrong place and (often) outside the map view area, if I zoom out to find the pin and click again on point X (without saving) the pin moves further north to again be out of the current (zoomed out) map view. Zoom out to find it and click point X again, red pin even further North to be out of the map view - and on and on until the Pin in hundreds of miles from point X.

I have tried with different photos and with photos from different cameras and with different countries selected in maps - still the same thing.

Basically the program appears broken.

I've attached a screen recording of this happening.

Hope you can get this sorted quickly.

https://github.com/marchyman/GeoTag/assets/172131039/5b30c44c-aaea-4e14-b452-c06cc8a6c0e3

**Comment by Orienteerer** [comment]
Note: I've just installed GeoTag v5.1 and this works as expected.

**Comment by marchyman** [comment]
I saw that when testing.  I coded a workaround for what was found to be an Apple error.  It lied when reporting the Y coordinate when converting clicks/taps to locations on the screen.  Apple fixed the error in either 14.4 or 14.4.1 and I removed my workaround.  Today I am running macOS 14.5 and do not see the issue.

So why does it work with version 5.1?  Versions 5.2.1 and earlier use an older version of Apple's Maps framework.  The bug was apparently in the newer framework and as noted above was fixed in an OS update.

If you can't update your OS I'd use version 5.2.1.  It is available at https://www.snafu.org/GeoTag/GeoTag-5.2.1.dmg
A better option in my opinion would be to update to the current version of macOS.

**Comment by Orienteerer** [comment]
Thanks. Your info also made me aware that my Mac has stopped notifying me and/or downloading OS updates!
